index.html

```diff
@@ -132,79 +132,106 @@
     <section class="informative">
         <h1>Examples</h1>
         <p>The following are example author code that demonstrates how the APIs in this specification might be used.</p>
-<pre id="example_1" class="example" title="Basic HTML5 Canvas painting application">
-&lt;style&gt;
+<pre id="example_1" class="example" title="Feature detection and event binding">
+<code>/* Bind to either Pointer Events or traditional touch/mouse */
+
+if (window.PointerEvent) {
+    // if Pointer Events are supported, only listen to pointer events
+    target.addEventListener("pointerdown", function(e) {
+        // if necessary, apply separate logic based on e.pointerType
+        // for different touch/pen/mouse behavior
+        ...
+    });
+    ...
+} else {
+    // traditional touch/mouse event handlers
+    target.addEventListener('touchstart', function(e) {
+        // prevent compatibility mouse events and click
+        e.preventDefault();
+        ...
+    });
+    ...
+    target.addEventListener('mousedown', ...);
+    ...
+}
+
+// additional event listeners for keyboard handling
+...</code>
+</pre>
+<pre id="example_2" class="example" title="Detecting the type of input from a user">
+<code>window.addEventListener("pointerdown", detectInputType, false);
```

Comment:
agree, this somehow snuck into the original and stayed around. unnecessary in modern browsers...i'll nuke it (and any other instances i find). good catch